sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/CopyAuthorizationResult.Serialization.cs

```diff
@@ -8,7 +8,7 @@
 using System.Text.Json;
 using Azure.Core;
 
-namespace Azure.AI.FormRecognizer.Training
+namespace Azure.AI.FormRecognizer.Models
```

Comment:
Like in the previous PR, the auto-generated types are always put in the Models namespace by default. These types are internal, so this is a considerably small price to pay for removing a whole file.